usecases/example1/HelloWorldSampleProject/K8sHelloWorldSample/config.properties

```diff
@@ -0,0 +1 @@
+containerType=kubernetes
```

Comment:
@sajithaliyanage please add an EOF line. Applies to all other, similar cases as well.

Reply:
Done